gluten-ut/spark34/src/test/scala/org/apache/spark/sql/sources/GlutenInsertSuite.scala

```diff
@@ -115,15 +115,15 @@ class GlutenInsertSuite
   }
 
   testGluten("Cleanup staging files if job is failed") {
-    withTable("t") {
-      spark.sql("CREATE TABLE t (c1 int, c2 string) USING PARQUET")
-      val table = spark.sessionState.catalog.getTableMetadata(TableIdentifier("t"))
+    withTable("t1") {
```

Comment:
Does the change fix the issue?

If yes it's weird as:

1. More `withTable('t')`s are in the file
2. `withTable('t') {}` should remove the files after scope exits

Reply:
@zhztheplayer, this is not the final fix. I suspect it's this test that causes following test failure, assuming its table is not removed somehow. But seems it's not.

Reply:
Aha I missed that the PR is in draft. Please go ahead. Thank.